### ISSUE TITLE
On Python 3.6 getting error

### ISSUE THREAD

**Issue by hnykda** [0xnurl/fado-python3#1]
```
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-5-505d8e45dd2f> in <module>()
----> 1 from FAdo.fa import *
      2 from FAdo.reex import *
      3 from FAdo.fio import *

/home/dan/.local/opt/miniconda3/envs/mathbase/lib/python3.6/site-packages/FAdo/fa.py in <module>()
     27 from __future__ import division
     28 from __future__ import unicode_literals
---> 29 from future import standard_library
     30 standard_library.install_aliases()
     31 from builtins import map

ModuleNotFoundError: No module named 'future'
```

**Comment by 0xnurl** [comment]
You need to install the `future` module: `$pip install future`

**Comment by hnykda** [comment]
Why it is not in setup.py?

**Comment by 0xnurl** [comment]
Fixed by 4252e02. Thanks!
